app/src/main/java/fi/bitrite/android/ws/ui/MapFragment.java

```diff
@@ -347,15 +343,18 @@ private void setGotoCurrentLocationStatus() {
                                            || mHasEnabledLocationProviders);
 
         int fillColor;
-        Drawable icon;
+        VectorDrawableCompat icon;
         if (mLastDeviceLocation.getValue() != null) {
-            icon = mIcMyLocationWhite;
+            icon = VectorDrawableCompat.create(getResources(), R.drawable.ic_my_location_white_24dp,
```

Comment:
Can we create these icons once in `onCreateView()` (or reset them to `null` in `onDestroyView()` and create on demand) s.t. we do not re-create them each time the location status changes.